mobile-tutorial/crafting.md

```diff
@@ -1,10 +1,11 @@
 ## Crafting
 
-Crafting is an important part of this game. With this function you can sustain yourself, supplying yourself with sufficient ammo and medical items. All resources needed can be looted from dungeon chests or scavenging. Crafting have independent level system, please refer to experience part of mobile tutorial for more information on these levels.
-Details on what you can craft and what materials you need are listed below.
+*Planning to add details about the various resources and how to craft them.*
 
 <br />
 
+Crafting can be found by going to `src > Player > Crafting.ts` for the Web version and `Inventory > Crafting` for Mobile players.
```

Comment:
Hey seems like this is for web version?

Reply:
I was lazy and put both ways haha, i can  change it up though

Reply:
Thanks 👍